great-idea-website/css/index.css

```diff
@@ -63,4 +63,157 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
```

Comment:
If you notice the image file, you see that the width of the page is rather narrow. You can achieve that effect by using a container `div` and using the properties `max-width`, `width`, and `margin`. Let me know if you want me to explain any of these properties in detail.